llarp/profiling.cpp

```diff
@@ -92,7 +96,8 @@ namespace llarp
   bool
   RouterProfile::IsGoodForPath(uint64_t chances) const
   {
-    return checkIsGood(pathFailCount, pathSuccessCount, chances);
+    return checkIsGood(pathFailCount, pathSuccessCount, chances)
```

Comment:
any reason `checkIsGood` isn't a member function?  Also, its name is very generic...

Reply:
`checkIsGood` is a constexpr because it's used across multiple different stats member variables.